composer/utils/file_helpers.py

```diff
@@ -345,6 +345,17 @@ def maybe_create_object_store_from_uri(uri: str) -> Optional[ObjectStore]:
     elif backend == 'wandb':
         raise NotImplementedError(f'There is no implementation for WandB load_object_store via URI. Please use '
                                   'WandBLogger')
+    elif backend == 'gs':
+        if 'GCS_KEY' not in os.environ or 'GCS_SECRET' not in os.environ:
```

Comment:
Are there standard environment variables we should be using for this? Like what does MCLOUD do for GCS auth? (What you have is probably correct, just checking)

Reply:
Yeah exactly streaming and mcli use those env variables